### ISSUE TITLE
bu fails if output directory does not exist

### ISSUE THREAD

**Issue by wonder-sk** [hobu/untwine#3]
When `bu` is run with an output directory that does not exist, it will abort:
```
terminate called after throwing an instance of 'pdalboost::filesystem::filesystem_error'
  what():  pdalboost::filesystem::create_directory: No such file or directory: "/tmp/bu/ept-data"
Aborted (core dumped)
```
Ideally it should auto-create the output directory.

**Comment by abellgithub** [comment]
This was by design, but can be changed.

**Comment by abellgithub** [comment]
Fixed in 0a8fab68964906976a56563bd83f1269f3893bc4

**Comment by abellgithub** [comment]
Seems to me like something the user can and should deal with.
